docs/development/minimal-reproductions.md

```diff
@@ -6,6 +6,15 @@ We may ask you to create a "minimal reproduction" repository.
 
 A minimal reproduction is a repository with the _least_ amount of code and config that still triggers missing or wrong behavior.
 
+### Example of a perfect reproduction
```

Comment:
I think showing the perfect example up front is nice, as readers can see how it's supposed to look, and then read the instructions.

But other readers may want to see the instructions first, and get going. 🙃

So let me know what you think! 😉

Reply:
I would prefer to even remove config:recommended, if it had nothing to do with the problem.

Reply:
I'm not sure, the original bug reporter had a repository `renovate.json` file with this:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "local>DanySK/renovate-config"
  ]
}
```

The `default.json` from that extended repository on 7th of October 2021 had this:

```json
{
  "extends": [
    "config:base", ":rebaseStalePrs"
  ],
  "assignees": ["DanySK"],
  "automerge": true,
  "dependencyDashboard": true,
  "git-submodules": {
    "enabled": true
  },
  "includeForks": true,
  "prConcurrentLimit": 25,
  "prHourlyLimit": 0,
  "separateMajorMinor": true,
  "separateMinorPatch": true,
  "separateMultipleMajor": true 
}
```

I guess that `config:base`/`config:recommended` was as minimal as I could make the reproduction at the time. In any case my goal was to have the "default user experience" in the reproduction.

As the bug is fixed now, I'm not sure how we can minimize the reproduction. :upside_down_face:

Reply:
Uninstall the bot (if necessary) then edit the config to be {}.

Reply:
The config is now `{}`.

Renovate is _not_ installed on the repository, at least as far as I can see.

The reproduction repository is _not_ a selected repository for any Renovate app in the `renovate-reproductions` organization. Here's how I checked: From the `renovate-reproductions organization's Settings tab -> Sidebar left, Third-Party Access section, select GitHub App -> select configure for each Renovate app.